### PR TITLE
Retrain model with another parameters

### PR DESCRIPTION
**Pull request by Yullia**
## Description

{"lstm_units": "122", "dense_1_units": "230", "activation": "relu", "learning_rate": 0.0001, "epochs": "18", "batch_size": "112"}

## How has this been tested?

Please describe the tests that you ran to verify your changes.
accuracy
loss

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

**Comment by babenek** [comment]
Thank you  for the PR. The experience was used in #610
